db/migrate/20211119104835_add_property_info_fields.rb

```diff
@@ -0,0 +1,13 @@
+class AddPropertyInfoFields < ActiveRecord::Migration[6.1]
+  def change
+    change_table :case_logs, bulk: true do |t|
+      t.column :do_you_know_the_postcode, :string
```

Comment:
Should these be `ints` (Yes/No enums) so it's consistent with how we're doing it for other fields?

Reply:
Done this for three fields